src/data/support-title.csv

```diff
@@ -1,65 +1,65 @@
 "text", "translation"
-"[トレセン学園]",""
-"[やれやれ、お帰り]",""
-"[努力は裏切らない！]",""
-"[テッペンに立て！]",""
-"[副会長の一刺し]",""
-"[デジタル充電中+]",""
-"[検証、開始]",""
-"[カワイイ＋カワイイは～？]",""
-"[雨の独奏、私の独創]",""
-"[鍛えぬくトモ]",""
-"[鍛えて、応えて！]",""
-"[シチーガール入門♯]",""
-"[生体Aに関する実験的研究]",""
-"[0500・定刻通り]",""
-"[波立つキモチ]",""
-"[マーベラス☆大作戦]",""
-"[運の行方]",""
-"[幸せと背中合わせ]",""
-"[目線は気にせず]",""
-"[…ただの水滴ですって]",""
-"[一流プランニング]",""
-"[共に同じ道を！]",""
-"[見習い魔女と長い夜]",""
-"[パリピ・ぱーりないと！]",""
-"[準備運動は怠るべからず]",""
-"[むじゃむじゃむじゃき]",""
-"[あなたにささげる]",""
-"[おすすめ本、あります！]",""
-"[世界の真珠、その名は]",""
-"[日本一のステージを]",""
-"[輝く景色の、その先に]",""
-"[夢は掲げるものなのだっ！]",""
-"[不沈艦の進撃]",""
-"[ロード・オブ・ウオッカ]",""
-"[千紫万紅にまぎれぬ一凛]",""
-"[パッションチャンピオーナ！]",""
-"[待望の大謀]",""
-"[天をも切り裂くイナズマ娘！]",""
-"[感謝は指先まで込めて]",""
-"[飛び出せ、キラメケ]",""
-"[B・N・Winner!!]",""
-"[7センチの先へ]",""
-"[Run(my)way]",""
-"[はやい！うまい！はやい！]",""
-"[一粒の安らぎ]",""
-"[これが私のウマドル道☆]",""
-"[まだ小さな蕾でも]",""
-"[うらら～な休日]",""
-"[必殺！Wキャロットパンチ！]",""
-"[ようこそ、トレセン学園へ！]",""
-"[『エース』として]",""
-"[『幸せ』が舞う時]",""
-"[『愛してもらうんだぞ』]",""
-"[夕焼けはあこがれの色]",""
-"[ターボエンジン全開宣言！]",""
-"[バカと笑え]",""
-"[迫る熱に押されて]",""
-"[その背中を越えて]",""
-"[Just keep going.]",""
-"[ふるさと直送エール！]",""
-"[押して忍べど燃ゆるもの]",""
-"[夢はホントに叶うんだ！]",""
-"[今ぞ盛りのさくら花]",""
-"[ふたつのノーブルライト]",""
+"[トレセン学園]","Tracen Academy"
+"[やれやれ、お帰り]","Oh Dear, Welcome Back"
```

Comment:
This doesn't have to necessarily be changed, more like a small nitpick... "Oh Dear, Welcome Back" sounds like she was welcoming her hubby back home, it doesn't really have the YAREYARE energy

Reply:
True enough. I didn't think about the double meaning of "Dear" here. Suggestions for alternatives?

Reply:
Everything just sounds so awkward here, I was thinking about something like "well, well", but "Well, well, welcome back" sounds more like she's stuttering lol

Reply:
"Good grief" sounds good enough imo